include/common_package/common_node.hpp

```diff
@@ -68,6 +74,36 @@ namespace common_lib
             RCLCPP_DEBUG(this->get_logger(), "CommonNode::deactivate: Deactivated node");
         }
 
+        /**
+         * @brief Handles the completion of a job.
+         *
+         * This function sends a job_finished message with the given error code and payload.
+         * Additionally, deactivates the node.
+         *
+         * @param error_code The error code associated with the job completion (EXIT_SUCCESS == 0 if no error).
+         * @param payload The payload data associated with the job completion.
+         */
+        void job_finished(const uint8_t error_code, const nlohmann::json payload);
```

Comment:
Die Payload könnte man als Referenz machen, im worst case kopiert er sonst die gesamte interne Datenstruktur, wenn es kein copy on write ist.
```suggestion
        void job_finished(const uint8_t error_code, const nlohmann::json &payload);
```

Reply:
Ist gemacht